crates/tx5-core/src/error.rs

```diff
@@ -8,6 +8,11 @@ pub struct Error {
     pub info: String,
 }
 
+impl From<Error> for () {
+    #[inline]
+    fn from(_: Error) -> Self {}
+}
+
```

Comment:
Yes, I sometimes use `Result<_, ()>` in unit tests. I don't think there's any way to accidentally loose information unless we have the `Result<_, ()>` type in actual code, which we don't. I'm inclined to leave this in here for my convenience, but could be convinced otherwise if you feel strongly about it.

Reply:
I couldn't find any instances of that in tx5. If it's for tests only and limited to the core crate, please put a `#[cfg(test)]` on it.

I'd like to see examples of how you use it. I can't fathom why it would be helpful to throw away an error, even during a test, since if a test fails you want to see why.

Reply:
*sigh* no convenience... I can delete it.

Reply:
done: https://github.com/holochain/tx5/pull/77/commits/95210064c7afde25dad5afbde55ae1c8a65d2f00